### PR TITLE
fix: styles on mobile for main nav

### PR DESCRIPTION
**Pull request by CzarekDryl**
https://tw.auroracreation.com/app/tasks/15787761

image

## Description

- Fixed UI styles for colony switcher on mobile

Resolves [1893](https://github.com/JoinColony/colonyCDapp/issues/1893)

### CODE REVIEW

**Review by warrencolliver — DISMISSED** [review]
Looks good to me - icons are now 18px and the padding sizes have been resolved.
Screenshot 2024-02-13 at 14 10 38

**Review by melyndav — CHANGES_REQUESTED** [review]
Hey @CzarekDryl thanks for picking up my issue. Everything looks good, just one minor change, can you match the user avatar height to the others in the top header?Should be 18px and looks like it's still 16px. Thanks! 

image

**Review by melyndav — APPROVED** [review]
Perfect @CzarekDryl! Smashed it. Approved from my side 👍

**Review by warrencolliver — APPROVED** [review]
Looks good - re approving

**Review by ArmandoGraterol — APPROVED** [review]
LGTM 👍

File: src/components/common/Extensions/UserNavigation/partials/HeaderAvatar/HeaderAvatar.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const HeaderAvatar = () => {
 
   return (
     <div className="flex items-center justify-center min-w-[2.625rem] min-h-[2.5rem] px-[0.875rem] py-[0.625rem] bg-base-white border rounded-full border-gray-200">
-      <UserAvatar user={user || wallet?.address} showUsername size="xxs" />
+      <UserAvatar user={user || wallet?.address} showUsername size="xss" />
```
Comment (rdig) [inline src/components/common/Extensions/UserNavigation/partials/HeaderAvatar/HeaderAvatar.tsx:19]: @CzarekDryl This change broke the user name display on the landing page

Screenshot from 2024-02-15 17-40-30
Screenshot from 2024-02-15 17-44-41

However, it does work fine when you're on a colony's home / dashboard

Screenshot from 2024-02-15 17-40-35

Please open a separate PR to fix this.

**Pull request merged**